HIRS_Provisioner.NET/hirs/appsettings.json

```diff
@@ -2,6 +2,7 @@
   "auto_detect_tpm":  "TRUE",
   "aca_address_port": "https://127.0.0.1:8443",
   "efi_prefix": "",
+  "cert_prefix": "",

```

Comment:
This cert_prefix option is confusing. efi_prefix is looking for the root of the EFI system partition. cert_prefix is looking for the exact directory to place files we generate. Consider renaming cert_prefix to better fit the value being requested. Example: certificate_output_directory